desktop/apps/auction2/works.js

```diff
@@ -2,11 +2,14 @@ import React from 'react'
 import ArtworksContainer from './components/auction_artworks'
 import HeaderContainer from './components/header'
 
-const Works = () => (
+export const Works = () => (
   <div>
-    <HeaderContainer />
     <ArtworksContainer />
   </div>
 )
 
-export default Works
+export const Header = () => (
+  <div>
+    <HeaderContainer />
+  </div>
+)
```

Comment:
One other thought -- if they're _extremely_ small, it might be a case of premature optimization and all of the markup can probably just exist inside of the container instead of breaking it out.